presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -590,7 +590,8 @@ The following table defines which temporal data types are supported by
 | ``TIMESTAMP``                       | ``custom-date-time``, ``iso8601``, ``rfc2822``,                                |
 |                                     | ``milliseconds-since-epoch``, ``seconds-since-epoch``                          |
 +-------------------------------------+--------------------------------------------------------------------------------+
-| ``TIMESTAMP WITH TIME ZONE``        | ``custom-date-time``, ``iso8601``, ``rfc2822``,                                |
+| ``TIMESTAMP WITH TIME ZONE``        | ``custom-date-time``, ``iso8601``, ``rfc2822``, ``milliseconds-since-epoch``,  |
```

Comment:
``TIME WITH TIME ZONE``   in this table?

Reply:
Early morning PRs ☕. The ``TIME`` row already has the proper entries.

Thanks for catching this.